sources/models/useful.js

```diff
@@ -0,0 +1,28 @@
+export const useful_functions = {
```

Comment:
this is a helper with different functions not connected to each other. 

so it is better to define and export them separately `export function smth..`

Reply:
ok

Reply:
they should be put in one file, but this is rather minor